src/tung_websocket.rs

```diff
@@ -377,8 +377,6 @@ impl<S: Unpin> Stream for TungWebSocket<S> where S: AsyncRead + AsyncWrite + Sen
 					// These are handshake errors:
 					//
 					TungErr::Url        (_)  |
-					TungErr::HttpFormat (_)  |
-					TungErr::Http       (_)  |
```

Comment:
anyway, I think you are asking these two, they were removed because these two need additional features to access

Reply:
https://github.com/snapview/tungstenite-rs/blob/e5efe537b87a6705467043fe44bb220ddf7c1ce8/src/error.rs#L67C22-L72

Reply:
@najamelan